packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -26,6 +24,16 @@ contract L2OutputOracle is OwnableUpgradeable, Semver {
      */
     uint256 public immutable L2_BLOCK_TIME;
 
+    /**
```

Comment:
We probably want to note that these can be changed by redeploying the contract and then changing the impl in the proxy